dask/dataframe/core.py

```diff
@@ -3735,6 +3735,7 @@ def resample(self, rule, closed=None, label=None):
 
         return Resampler(self, rule, closed=closed, label=label)
 
+    @_deprecated(message="Will be removed in a future version.")
```

Comment:
Can you add messages what users should do instead? You can check the pandas docs for a guide

Reply:
https://github.com/dask/dask/pull/10723/commits/287c4b571d0fafdc94a0c55df5824d6819dbf391 that's better. :+1: